docling/backend/docx/latex/omml.py

```diff
@@ -281,8 +281,10 @@ def do_fname(self, elm):
                 if FUNC.get(t):
                     latex_chars.append(FUNC[t])
                 else:
-                    raise NotSupport("Not support func %s" % t)
-            else:
+                    print(f"Function not supported, will default to text: {t}")
```

Comment:
Let's remove this print

Reply:
I replaced the print by a logger warning. It's still useful to log these instances, as we might want to add them to the latex dict.